test-e2e/build/fixtures/no-deps-backslash/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "no-deps-backslash",
+  "version": "1.0.0",
+  "license": "MIT",
+  "esy": {
+    "build": [
+      [
+        "bash",
+        "-c",
+        "echo '#!/bin/bash\necho \\\\\\\\ #{self.name}  \\\\\\\\' > #{self.target_dir / self.name}"
```

Comment:
@andreypopp - I think this exercises the case we were looking at. Without the reversion in `ShellParamExpansion.ml` above, these would get flipped to `//`.

Note the escaping behavior:
- When running the `bash` command the `\\\\\\\\` gets escaped to `\\\\`
- When the `\\\\ #{self.name} \\\\` gets echo'd, it gets escaped to `\\ #{self.name} \\`
(this then outputs `\ no-deps-backslash \`.